install-worker.sh

```diff
@@ -227,6 +227,16 @@ EOF
 sudo mv /tmp/release /etc/eks/release
 sudo chown root:root /etc/eks/*
 
+################################################################################
+### Stuff required by "protectKernelDefaults=true" #############################
+################################################################################
+
+cat <<EOF | sudo tee -a /etc/sysctl.d/99-amazon.conf
+vm.overcommit_memory=1
+kernel.panic=10
```

Comment:
Any reason to pick 10 here? I was trying to learn more about these config params and ran into these docs

```
The value in this file represents the number of seconds the kernel
waits before rebooting on a panic. When you use the software watchdog,
the recommended setting is 60.
```
https://www.kernel.org/doc/Documentation/sysctl/kernel.txt

Reply:
Looks like kubernetes expects it to be 10 seconds - https://github.com/kubernetes/kubernetes/blob/d8695d06b7191db56ebbbc0340da263833c9bb6f/pkg/util/sysctl/sysctl.go#L56

@abeer91 Given that I'd need to have this merged. Would you want me to find out why k8s wants exactly 10 second timeout?